<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1411</width>
      <height>858</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des employées</string>
     </attribute>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>670</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Ajouter.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>286</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mail">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>618</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adr">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>542</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numtel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>479</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>670</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Modifier.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>670</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Supprimer.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="affichage">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>40</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Afficher.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="chercher">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>40</y>
        <width>132</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Rechercher.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="cherchertext">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>40</y>
        <width>151</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="chercherchoix">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>151</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/LD.png);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Identifiant Employé</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Numéro de télèphone</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tabemp">
      <property name="geometry">
       <rect>
        <x>569</x>
        <y>106</y>
        <width>819</width>
        <height>417</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="photoEmp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1411</width>
        <height>858</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="la_dns">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>415</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>photoEmp</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>le_prenom</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_mail</zorder>
     <zorder>le_id</zorder>
     <zorder>le_adr</zorder>
     <zorder>le_numtel</zorder>
     <zorder>modifier</zorder>
     <zorder>supprimer</zorder>
     <zorder>affichage</zorder>
     <zorder>chercher</zorder>
     <zorder>cherchertext</zorder>
     <zorder>chercherchoix</zorder>
     <zorder>tabemp</zorder>
     <zorder>la_dns</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des congés</string>
     </attribute>
     <widget class="QLineEdit" name="le_prenomc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>328</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_idc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>201</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>658</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nomc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>264</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="deb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>391</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="etat">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>590</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="fin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>453</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="dur">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>520</y>
        <width>127</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>610</y>
        <width>184</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/envoyermail.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifierc">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>570</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Modifier.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimerc">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>630</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Supprimer.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouterc">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>510</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Ajouter.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="affichagec">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>40</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Afficher.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="choixR">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>151</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/LD.png);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Identifiant Congée</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Durée</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="recherchec">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>40</y>
        <width>132</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Rechercher.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="choix">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>40</y>
        <width>151</width>
        <height>38</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabconge">
      <property name="geometry">
       <rect>
        <x>569</x>
        <y>106</y>
        <width>819</width>
        <height>417</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TrierC">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>550</y>
        <width>120</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/Trier.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="choixT">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>550</y>
        <width>151</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Junior Prince</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/photo/img/LD.png);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Identifiant Congée</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Durée</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1411</width>
        <height>858</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="photoCong">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1411</width>
        <height>858</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>photoCong</zorder>
     <zorder>label</zorder>
     <zorder>le_prenomc</zorder>
     <zorder>le_idc</zorder>
     <zorder>le_id_2</zorder>
     <zorder>le_nomc</zorder>
     <zorder>deb</zorder>
     <zorder>etat</zorder>
     <zorder>fin</zorder>
     <zorder>dur</zorder>
     <zorder>pushButton_27</zorder>
     <zorder>pb_modifierc</zorder>
     <zorder>pb_supprimerc</zorder>
     <zorder>pb_ajouterc</zorder>
     <zorder>affichagec</zorder>
     <zorder>choixR</zorder>
     <zorder>recherchec</zorder>
     <zorder>choix</zorder>
     <zorder>tabconge</zorder>
     <zorder>TrierC</zorder>
     <zorder>choixT</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
